pkgs/test/stdenv/default.nix

```diff
@@ -99,6 +99,8 @@ let
 in
 
 {
+  __attrsFailEvaluation = stdenv.isDarwin;
```

Comment:
I'm not the most pleased this entire suite is turned off on Darwin. It's ok if necessary but it's a little unfortunate.

Reply:
Yeah, it had a funky error without it.

Reply:
This shouldn't be an issue, idk why it's a problem. I kinda didn't feel like diving into figuring out why this was happening so the easiest solution was to just disable on Darwin. Linux targets work so this is a Darwin only issue. If this isn't a blocker and we can't find a solution before this PR is merged, a follow up PR to properly fix it should be fine.
```
error:
       … while calling the 'map' builtin

         at /home/ross/ExpidusOS/nixpkgs/pkgs/top-level/release-attrpaths-superset.nix:193:5:

          192|   names =
          193|     map (path: (lib.concatStringsSep "." path)) paths;
             |     ^
          194|

       … while evaluating a branch condition

         at /home/ross/ExpidusOS/nixpkgs/pkgs/top-level/release-attrpaths-superset.nix:162:9:

          161|       result =
          162|         if tried.success
             |         ^
          163|         then tried.value

       (stack trace truncated; use '--show-trace' to show the full trace)

       error: attribute '__bootPackages' missing

       at /home/ross/ExpidusOS/nixpkgs/pkgs/test/stdenv/default.nix:13:17:

           12|   earlyPkgs = stdenv.__bootPackages.stdenv.__bootPackages;
           13|   earlierPkgs = stdenv.__bootPackages.stdenv.__bootPackages.stdenv.__bootPackages.stdenv.__bootPackages.stdenv.__bootPackages;
             |                 ^
           14|   # use a early stdenv so when hacking on stdenv this test can be run quickly
```

Reply:
there's got to be a way to fix this better because `__bootPackages` is certainly not the only attr that is broken in the entirety of nixpkgs, right?? but can you add a FIXME above that disabling, if you want to defer it?

Reply:
also cc @reckenrode as a darwin maintainer noting that this was deferred.

Reply:
> but can you add a FIXME above that disabling, if you do want to defer it; with that change this is lgtm without figuring this out.

Will do.

Reply:
Added the note.